docs/examples/movie-reviews-graph-rag.ipynb

```diff
@@ -0,0 +1,568 @@
+{
```

Comment:
<p>This pattern is hard to use in colab (you need to create a file). In our other examples, we've used a utility that automatically attempts to load from environment (if possible), env file (if present), colab secrets (if running in colab), and falls back to prompting the user. It may be worth using that?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/datastax/graph-rag/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>